tests/covalent_tests/workflow/lattice_serialization_test.py

```diff
@@ -35,7 +35,7 @@ def workflow(x):
         return f(x)
 
     workflow.build_graph(5)
-    workflow.cova_imports = set("dummy_module")
+    workflow.cova_imports = set({"dummy_module"})
```

Comment:
This automatically becomes a set if you just use `{}`, no type conversion needed.